src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -100,30 +99,29 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
             ${supportsFeature(stateObj, SUPPORT_BRIGHTNESS)
               ? html`
                   <round-slider
-                    min="1"
+                    min="0"
                     .value=${brightness}
                     @value-changing=${this._dragEvent}
                     @value-changed=${this._setBrightness}
                   ></round-slider>
                 `
               : ""}
-            <ha-icon
-              class="slider-center"
-              data-state="${stateObj.state}"
-              .icon="${this._config.icon || stateIcon(stateObj)}"
-              style="${styleMap({
+            <paper-icon-button
+              class="light-button ${classMap({
+                "slider-center": supportsFeature(stateObj, SUPPORT_BRIGHTNESS),
+              })}"
+              .icon=${this._config.icon || stateIcon(stateObj)}
+              style=${styleMap({
                 filter: this._computeBrightness(stateObj),
                 color: this._computeColor(stateObj),
-              })}"
-              @action="${this._handleClick}"
-              .actionHandler=${actionHandler()}
-              tabindex="0"
-            ></ha-icon>
+              })}
+              @click=${this._handleClick}
+            ></paper-icon-button>
           </div>
         </div>
 
         <div id="info">
-          <div class="brightness" @ha-click="${this._handleClick}">
+          <div class="brightness">
```

Comment:
Good!
I couldn't remember why this was here, so I left it in. But I really can't see any point to it at all.